administrator/modules/mod_community_info/services/provider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_community_info
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Extension\Service\Provider\HelperFactory;
+use Joomla\CMS\Extension\Service\Provider\Module;
+use Joomla\CMS\Extension\Service\Provider\ModuleDispatcherFactory;
+use Joomla\DI\Container;
+use Joomla\DI\ServiceProviderInterface;
+
+/**
+ * The community info module service provider.
+ *
+ * @since  4.5.0
+ */
+return new class () implements ServiceProviderInterface {
+    /**
+     * Registers the service provider with a DI container.
+     *
+     * @param   Container  $container  The DI container.
+     *
+     * @return  void
+     *
+     * @since   4.5.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```
Repeat with other instances.

Reply:
Done!